website/examples/keyboard.test.tsx

```diff
@@ -199,3 +205,35 @@ describe.each(['ltr', 'rtl'])(
     });
   }
 );
+
+describe('keyboard actions', () => {
+  describe('when the Page Up is pressed', () => {
```

Comment:
All these `describes` should go inside the first `describe`, as they must run the `setup` before each and test the rtf direction too.